### PR TITLE
Fixed some issues in the readme

### PR DESCRIPTION
**Pull request by CW668**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
...
-## Members
-Connor Wood (cw668),
-Dean Gooding (dag2),
-Bethany Hill (berh2),
-William Haysom (wah20),
-Jack Fermer (jf500).
\ No newline at end of file
+- Connor Wood (cw668)
+- Dean Gooding (dag2)
+- Bethany Hill (berh2)
```
Comment (berh2) [inline README.md:5]: excuse me but why am I third?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,7 @@
-# Chop Chop!
+# Chop Chop!
```
Comment (dmag2) [inline README.md:1]: Remove This

File: README.md [hunk 3]
```diff
@@ -1,8 +1,7 @@
-# Chop Chop!
+# Chop Chop!  
 
-## Members
-Connor Wood (cw668),
-Dean Gooding (dag2),
-Bethany Hill (berh2),
-William Haysom (wah20),
-Jack Fermer (jf500).
```
Comment (dmag2) [inline README.md:8]: Unneeded

**Review by JF500 — COMMENTED** [review]
approved

**Review by dmag2 — COMMENTED** [review]
Overall pretty good, merge this in : )

File: README.md (hunk 1 above)
Comment (dmag2) [reply README.md:5]: sounds like a skill issue

**Review by wah200 — APPROVED** [review]
looks shit but it's okay for Connor

File: README.md [hunk 4]
```diff
@@ -1,8 +1,7 @@
...
\ No newline at end of file
+- Connor Wood (cw668)
+- Dean Gooding (dag2)
+- Bethany Hill (berh2)
+- William Haysom (wah20)
+- Jack Fermer (jf500)
```
Comment (JF500) [inline README.md:7]: this needs to be higher up

**Review by dmag2 — APPROVED** [review]
Well done, proud of you : *

**Review by berh2 — APPROVED** [review]

**Review by JF500 — APPROVED** [review]
lmao

**Pull request merged**